deploy/operator/hypershift/deploy_hypershift_cluster.sh

```diff
@@ -121,3 +121,35 @@ echo "Cluster installation has been stopped (either for good or bad reasons)"
 
 KUBECONFIG=$SPOKE_KUBECONFIG wait_for_condition "agentclusterinstall/${ASSISTED_AGENT_CLUSTER_INSTALL_NAME}" "Completed" "90m" "${SPOKE_NAMESPACE}"
 echo "Cluster has been installed successfully!"
+
+
+# Test webhooks
+
+function validate_webhook() {
```

Comment:
consider renaming, because it does something a bit more generalized
maybe ``search_pattern_in_line``? or ``validate_pattern_in_line``

Reply:
hmm, yeah maybe just validate_by_pattern is clear enough